src/main/java/com/google/step/datamanager/CommentManager.java

```diff
@@ -0,0 +1,14 @@
+package com.google.step.datamanager;
+
+import com.google.step.model.Comment;
+import java.util.List;
+
+public interface CommentManager {
+  public Comment createComment(long dealId, long userId, String content);
+
+  public List<Comment> getComments(long dealId);
```

Comment:
Name this `getCommentsByDealId` or `getCommentsForDeal`. I would usually expect `getComments` to take comment ids.